src/components/InstructorStatsCard/index.js

```diff
@@ -14,18 +14,28 @@ export default ({publishedLessons, publishedCourses}) => {
   const items = [
     {
       type: 'lesson',
-      text: `${publishedLessons} published lessons`,
+      text: publishedLessons === 1
+        ? <Text message='instructorStats.lessonsSingular' />
+        : <Text 
+            message='instructorStats.lessons'
+            values={[publishedLessons]} 
+          />,
       color: 'green',
     },
     {
       type: 'course',
-      text: `${publishedCourses} published courses`,
+      text: publishedCourses === 1
+        ? <Text message='instructorStats.coursesSingular' />
+        : <Text 
+            message='instructorStats.courses'
+            values={[publishedCourses]} 
+          />,
```

Comment:
Adds pluralization
<img width="481" alt="pluralization" src="https://cloud.githubusercontent.com/assets/5497885/24312521/a1219862-109e-11e7-985e-018225c9c71b.png">
From way back when  you mentioned this @tayiorbeii 👍